config/plugin_config.go

```diff
@@ -14,7 +35,7 @@ type PluginConfig interface {
 	GetValue(data interface{}) (found bool, err error)
 
 	// GetConfigName returns usually derived config name from plugin name
-	// PluginName + "-config"
+	// PluginName + "-config" (most often absolute path to a config file)
```

Comment:
Please explain in the comment what PluginName + "-config" means (it seems as if that was the config file name). Explain the use of the flag to construct the config file path.